AlpheusCore/API.fs

```diff
@@ -196,6 +196,12 @@ let saveAsync (experimentRoot, artefactId) storageName saveAll =
             let save = StorageFactory.getStorageSaver experimentRoot storageToSaveTo
             let saveDescriptors = artefactsToSave |> Array.map (fun art -> (idToFullPath experimentRoot artefactId),art.ActualHash.Value)
             let! _ = save saveDescriptors
+
+            // adding the saved artefact into the gitignore
+            let newIgnoreEntries = artefactsToSave |> Array.map (fun art -> art.Id.ToString())
```

Comment:
`ArtefactId.ToString()` potentially can be an arbitrary string. I suggest using the following:

```FSharp
open PathUtils
let gitIgnoreFolder = Path.GetDirectoryName(gitIgnorePath)

let newIgnoreEntry art = art |> idToFullPath experimentRoot |> relativePath gitIgnoreFolder |> unixPath
```

Reply:
Agree